scripts/validate_docstrings.py

```diff
@@ -679,5 +760,24 @@ def header(title, width=80, char='#'):
                            nargs='?',
                            default=None,
                            help=func_help)
+    argparser.add_argument('--format', default='default', help='format of the '
```

Comment:
I think you can pass `choices=format_opts` here and argparse does the validation for you.